packages/react/src/components/Modal/ModalPlumbing.tsx

```diff
@@ -1,71 +1,32 @@
-import { BottomSheet, ModalContext, ModalTitle } from '.'
-import styled, { css } from 'styled-components'
-import { useContext } from 'react'
-import { maxWidth } from '@charcoal-ui/utils'
+import { ModalContext } from '.'
+import { forwardRef, useContext } from 'react'
+import { useClassNames } from '../../_lib/useClassNames'
+
+import './ModalPlumbing.css'
 
 export function ModalHeader() {
   const modalCtx = useContext(ModalContext)
   return (
-    <ModalHeaderRoot $bottomSheet={modalCtx.bottomSheet}>
-      <StyledModalTitle />
-    </ModalHeaderRoot>
+    <div
+      className="charcoal-modal-header-root"
+      data-bottom-sheet={modalCtx.bottomSheet}
+    >
+      <div className="charcoal-modal-header-title">{modalCtx.title}</div>
+    </div>
   )
 }
 
-const ModalHeaderRoot = styled.div<{
-  $bottomSheet: BottomSheet
-}>`
-  height: 64px;
-  display: grid;
-  align-content: center;
-  justify-content: center;
-  @media ${({ theme }) => maxWidth(theme.breakpoint.screen1)} {
-    ${(p) =>
-      p.$bottomSheet !== false &&
-      css`
-        height: 48px;
-      `}
-  }
-`
-
-const StyledModalTitle = styled(ModalTitle)`
-  color: var(--charcoal-text1);
-  font-size: 16px;
-  line-height: 24px;
-  font-weight: bold;
-  display: flow-root;
-
-  &::before {
-    display: block;
-    width: 0;
-    height: 0;
-    content: '';
-    margin-top: -4px;
-  }
-  &::after {
-    display: block;
-    width: 0;
-    height: 0;
-    content: '';
-    margin-bottom: -4px;
-  }
-`
+export const ModalAlign = createDivComponent('charcoal-modal-align')
 
-export const ModalAlign = styled.div`
-  padding-left: 16px;
-  padding-right: 16px;
-`
+export const ModalBody = createDivComponent('charcoal-modal-body')
 
-export const ModalBody = styled.div`
-  padding-bottom: 40px;
-`
+export const ModalButtons = createDivComponent('charcoal-modal-buttons')
 
-export const ModalButtons = styled.div`
-  display: grid;
-  grid-auto-flow: row;
-  grid-row-gap: 8px;
-
-  padding-top: 16px;
-  padding-left: 16px;
-  padding-right: 16px;
-`
+function createDivComponent(mainClassName: string) {
+  return forwardRef<HTMLDivElement, React.ComponentPropsWithRef<'div'>>(
+    function ModalBody({ className, ...props }, ref) {
```

Comment:
suggest renaming this component to `DivComponent`.

```suggestion
    function DivComponent({ className, ...props }, ref) {
```

Reply:
Thanks for the suggestion. I've fixed this.